dev-ml/libvirt-ocaml/libvirt-ocaml-0.6.1.7.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="OCaml language binding for libvirt native C API"
+
+HOMEPAGE="https://ocaml.libvirt.org/"
+SRC_URI="https://download.libvirt.org/ocaml/ocaml-libvirt-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Let's avoid the renaming of the distfile (drop the `-> ...`). It harms provenance for no reason.

Reply:
Dropped the rename